backend-shared/config/kcp/apiexport_gitopsrvc.yaml

```diff
@@ -0,0 +1,14 @@
+# This file is generated from CRDs by ./hack/generate-kcp-api.sh script.
+# Please do not modify!
+
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: gitopsrvc
+spec:
+  latestResourceSchemas:
```

Comment:
Do we need to include [permissionClaims](https://github.com/kcp-dev/kcp/blob/6d2008e2fdeebbb3ef7874e7d1a559fca4e1ac53/pkg/apis/apis/v1alpha1/types_apiexport.go#L157) for secrets and other resources? I'm hitting this error when testing it with controllers


```shell
2022-07-28T06:05:23.610Z    ERROR    controller-runtime.source    source/source.go:137    if kind is a CRD, it should be installed before calling Start    {"kind": "Secret", "error": "no matches for kind \
"Secret\" in version \"v1\""}
```

Reply:
As per the findings, it is agreed to make the gitops-service controller happy we need at least namespaces secrets to get this working, but to address this another problem I'm referencing this issue #198 here as well (separate from the 188 stories)